theseus/utils/examples/motion_planning/motion_planner.py

```diff
@@ -53,6 +53,57 @@ def _copy_impl(self, new_name: Optional[str] = None) -> "_XYDifference":
         )
 
 
+class Nonholonomic(th.CostFunction):
```

Comment:
Maybe we should move this to embodied/motionmodel/

Reply:
Moved in 1918a240e39eed6b2186fd3c3bc17c860a445e75.